Document project resources section of contribution guide (#875)
submariner-io/submariner-website: src/content/development/contribution-guide/_index.en.md

## Project Resources

> Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on your Project Board] to learn more on how to work with

Reviewer: ```suggestion
Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on your Project Board] to learn more about how to work with
```

projects.

### Backlog Board

[The backlog board] hosts issues, bugs and features which are not a part of a particular release board. While each issue is opened in
its corresponding repository, this board gives an aggregated view of all open issues across all repositories. The board has total of seven

> columns. Any issue or epic opened should be assigned to `Backlog` Projects in `Backlog` column. Epics which are candidates to be included in

Reviewer: ```suggestion
columns. Any new issues or epics should typically be assigned to the `Backlog` column. Epics which are candidates to be included in
```

the next release are moved to the `Next Version Candidate` column. During [planning](#planning), all epics from the `Next

> Version Candidate` column are getting review and triage. Epics which are selected to the next release are moved to

Reviewer: ```suggestion
Version Candidate` column are reviewed and triaged. Epics which are selected to be in the next release are moved to
```

> [current release board](#current-release-board). The next 3 columns sorts issues by priority. All unassigned issues are sorted into these

Reviewer: ```suggestion
the [current release board](#current-release-board). The next 3 columns sort issues by priority. All unassigned issues are sorted into these
```

three columns based on the priority discussed in the team meeting. Issues from these columns are ready to be worked on. Any leftover work

> from the previous release goes to the `Work in Progess column`. `Close?` column holds any issue/epic that could probably be closed for

Reviewer: ```suggestion
from the previous release goes to the `Work in Progress column`. The `Close?` column holds any issues/epics that can probably be closed for
```

> from the previous release goes to the `Work in Progress column`. The `Close?` column holds any issues/epics that can probably be closed for

Reviewer 2: > The `Close?` column...

Are you waiting for any inputs from someone?

Author: I just wanted to give some time to folks to review. I think we can merge now

Reviewer 2: Okay. So do you want to remove the "?"

Author: This is the name of the column on the Backlog board - `Close?`, so it included in the description along side with the other columns description

various reasons.

### Current Release Board

Current release work is tracked in [the latest board]. The board has 5 columns. The first one, `Schedule and Epics`, hosts all properly
triaged epics targeted for the current release and the schedule for the current release. Triaged issues are under the `To do` column. When
an issue is being worked on, it is moved to `In Progress` column. When a PR for the issue is pushed, the issue is moved to `In Review`
column. The PR has the corresponding issue linked from the GitHub UI and is not tracked on the board. Once the PR is merged, the issue
is moved to `Done` column.

### Enhancements Repository

> The [Enhancements repository] in used for tracking [epics](#epics) as issues, and their corresponding enhancement proposals (the epics design)

Reviewer: ```suggestion
The [Enhancements repository] is used for tracking [epics](#epics) as issues, and their corresponding enhancement proposals (the epics design)
```

> as [pull requests to the enhancements repository]. Any medium or small enhancement should be opened as an issue to the relevant repository,

Reviewer: ```suggestion
as [pull requests to the enhancements repository]. Any medium or small enhancement should be opened as an issue in the relevant repository,
```

> or as a general [enhancement Request issue].

Reviewer: ```suggestion
or as a general [Enhancement Request issue].
```
